app/models/manual.rb

```diff
@@ -350,5 +350,10 @@ def set(attributes = {})
     manual_record.set(attributes)
   end
 
+  def link_check_report
+    link_check_reports = LinkCheckReport.where(manual_id: id)
+    link_check_reports ? link_check_reports.first : nil
```

Comment:
Can we memoize this?

Reply:
Won't `link_check_reports` always be non-`nil` because it's a scope of records?

Would this work `LinkCheckReport.find_by(manual_id: id)` instead of these two lines?